Makefile

```diff
@@ -22,11 +22,10 @@ calicobuild.created: $(BUILD_FILES)
 	cd build_calicoctl; docker build -t calico/build-libnetwork .
 	touch calicobuild.created
 
-calicoctl:
-	#wget https://github.com/projectcalico/calico-docker/releases/download/v0.5.5/calicoctl
-	# Update to a release when there is one.
-	wget https://circle-artifacts.com/gh/projectcalico/calico-docker/1860/artifacts/0/home/ubuntu/calico-docker/dist/calicoctl
-	chmod +x calicoctl
+dist/calicoctl:
+	mkdir dist
+	curl -L https://github.com/projectcalico/calico-docker/releases/download/v0.10.0/calicoctl -o dist/calicoctl
```

Comment:
I'll change to the latest link, but see above - we don't need to be using the latest. Calicoctl and calico-node code have very little impact on libnetwork-plugin (note that we were using the exact same calicoctl for months...)